zeroeq/http/server.h

```diff
@@ -258,28 +263,31 @@ class Server : public zeroeq::Receiver
      * @param func the callback function for serving the GET request
      */
     ZEROEQHTTP_API
-    bool handleGET( const std::string& endpoint, const std::string& schema,
-                    const GETFunc& func );
+    bool handleGET(const std::string& endpoint, const std::string& schema,
+                   const GETFunc& func);
 
     /**
      * @return the registered schema for the given object, or empty if not
      *         registered.
      */
     ZEROEQHTTP_API
-    std::string getSchema( const servus::Serializable& object ) const;
+    std::string getSchema(const servus::Serializable& object) const;
 
     /** @overload */
-    ZEROEQHTTP_API std::string getSchema( const std::string& endpoint ) const;
+    ZEROEQHTTP_API std::string getSchema(const std::string& endpoint) const;
     //@}
 
 private:
     class Impl;
-    std::unique_ptr< Impl > _impl;
+    std::unique_ptr<Impl> _impl;
 
     // Receiver API
-    void addSockets( std::vector< detail::Socket >& entries ) final;
-    void process( detail::Socket& socket, uint32_t timeout ) final;
-    void addConnection( const std::string& ) final { ZEROEQDONTCALL; } // LCOV_EXCL_LINE
+    void addSockets(std::vector<detail::Socket>& entries) final;
+    void process(detail::Socket& socket, uint32_t timeout) final;
+    void addConnection(const std::string&) final
+    {
+        ZEROEQDONTCALL;
+    } // LCOV_EXCL_LINE
```

Comment:
you get a false positive in the coverage report then. zeroeq had almost 100%, so then we can also disable coverage report generation and save 200 lines of cmake code